src/test/java/org/example/CustomListPerformanceTest.java

```diff
@@ -113,28 +195,77 @@ private void printResults(Result[] results, Result customListResult) {
-        Result customListResultObj = testObjectBulkAddition("CustomList", new CustomList<>(), BULK_SIZE);
+        System.out.printf("\n--- %s (%,d objects) ---\n", additionTestName, BULK_SIZE);
+        Result customListResultObj = testAddObjectBulkAddition("CustomList", new CustomList<>(), BULK_SIZE);
         Result[] bulkResultsObj = new Result[]{
-                testObjectBulkAddition("LinkedList", new LinkedList<>(), BULK_SIZE),
-                testObjectBulkAddition("ArrayList", new ArrayList<>(), BULK_SIZE)
+                testAddObjectBulkAddition("ArrayList", new ArrayList<>(), BULK_SIZE),
+                testAddObjectBulkAddition("LinkedList", new LinkedList<>(), BULK_SIZE),
+                testAddObjectBulkAddition("CustomLinkedList", new CustomLinkedList<>(), BULK_SIZE),
+                testAddObjectBulkAddition("Stack", new Stack<>(), BULK_SIZE),
+                testAddObjectBulkAddition("CustomStack", new CustomStack<>(), BULK_SIZE),
+                testAddObjectBulkAddition("CustomQueue", new CustomQueue<>(), BULK_SIZE)
         };
         printResults(bulkResultsObj, customListResultObj);
 
-        System.out.println("\n--- Add/Remove Test (10,000 add, 10,000 remove) ---");
+        System.out.printf("\n--- %s (%,d add, %,d remove) ---\n", addRemoveTestName, ADD_REMOVE_SIZE, ADD_REMOVE_SIZE);
         Result customListAddRemoveResult = testAddRemove("CustomList", new CustomList<>(), ADD_REMOVE_SIZE);
         Result[] addRemoveResults = new Result[]{
+                testAddRemove("ArrayList", new ArrayList<>(), ADD_REMOVE_SIZE),
                 testAddRemove("LinkedList", new LinkedList<>(), ADD_REMOVE_SIZE),
-                testAddRemove("ArrayList", new ArrayList<>(), ADD_REMOVE_SIZE)
+                testAddRemove("CustomLinkedList", new CustomLinkedList<>(), ADD_REMOVE_SIZE),
+                testAddRemove("Stack", new Stack<>(), ADD_REMOVE_SIZE),
+                testAddRemove("CustomStack", new CustomStack<>(), ADD_REMOVE_SIZE),
+                testAddRemove("CustomQueue", new CustomQueue<>(), ADD_REMOVE_SIZE)
+        };
+        printResults(addRemoveResults, customListAddRemoveResult);
+    }
+
+    @Test
+    @ThisIsTest
+    void altAddPerformanceReport() {
+        String additionTestName = "Bulk Alt Addition Test using dedicated methods";
+        String addRemoveTestName = "Alt Add/Remove Test using dedicated methods";
+
+        System.out.printf("\n--- %s (%,d integers) ---\n", additionTestName, BULK_SIZE);
+        Result customListResultInt = testAddIntBulkAddition("CustomList", new CustomList<>(), BULK_SIZE);
+        Result[] bulkResultsInt = new Result[]{
+                testPushIntBulkAddition("CustomLinkedList", new CustomLinkedList<>(), BULK_SIZE),
+                testPushIntBulkAddition("CustomStack", new CustomStack<>(), BULK_SIZE),
+                testEnqueueIntBulkAddition("CustomQueue", new CustomQueue<>(), BULK_SIZE),
```

Comment:
@ParametrizedTest ?

Reply:
I wanted to but since these are the custom test decorators and I did not create a custom decorator for parameterization I forwent this. I will get back to this feature